### PR TITLE
Added feature to press a button in order to load more past meetups

### PR DESCRIPTION
**Pull request by mzampetakis**
Initially we load 10 past meetups and then we load more meetups by
pressing a 'Load more' button. The meetups are loaded in a bunch
of 10.

### CODE REVIEW

**Comment by mzampetakis** [comment]
Any comments on this? Is this something we need but should be implemented otherwise?
Is it something we don't need at all?

**Comment by sonaht** [comment]
@kouts can you please check, review and let me know to merge or merge ahead pls.

File: index.html [hunk 1]
```diff
@@ -160,6 +162,11 @@ <h4><b>{{ meetup.Day }}</b>/{{ meetup.Month }}/{{ meetup.Year }}<br><b>{{ meetup
           </a>
         </div>
       </div>
+      <div class="text-center" ng-show="totalPastMeetups() > loadedMeetups && $last!=true && $index==loadedMeetups">
+        <button type="button" class="btn btn-default btn-sm" ng-click="loadMoreMeetups()">
+          <span class="glyphicon glyphicon-refresh"></span> More Events
```
Comment (kouts) [inline index.html]: For some reason ```glyphicon glyphicon-refresh``` on this span doesn't get displayed as it should.
Maybe it's a font or a css issue, can you check please?

**Comment by mzampetakis** [comment]
Thanks @kouts! Please take a look on the new corrections!

**Pull request merged**